pkgs/development/python-modules/pyialarm/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, dicttoxml
+, fetchFromGitHub
+, pythonOlder
+, xmltodict
```

Comment:
Sorting
```suggestion
{ lib
, buildPythonPackage
, pythonOlder
, fetchFromGitHub
, dicttoxml
, xmltodict
```

Reply:
Sorry, I do the sorting alphabetically after `lib` till there is an other reproducible way to do it automatically or a guideline that is compulsory. 

Pretty much all "custom sorting approaches" have mismatches after a short time. IMHO, grouping something that's already grouped doesn't make much sense.

Reply:
Yeah, it's not incorrect, hence the approval. I think it makes sense to me logically to see arguments grouped roughly by how they're used, which was helpful for me when looking at example packages when getting started, but nothing wrong with your way.

Reply:
I would love to have something like `isort` that is hooked into `nixpkgs-fmt` for the input.